fdbserver/VersionedBTree.actor.cpp

```diff
@@ -7359,8 +7444,8 @@ class VersionedBTree {
 											PhysicalPageID newID =
 											    self->m_pager->detachRemappedPage(p, batch->writeVersion);
 											if (newID != invalidPhysicalPageID) {
-												// Records in the rebuild vector reference original page memory so make
-												// a new child link in the rebuild arena
+												// Records in the rebuild vector reference original page memory so
+												// make a new child link in the rebuild arena
```

Comment:
looks like I introduced some unrelated format change. hope it don't hurt the review experience too much. I will revert the format change later.